src/scripts/circleci-job-canceller/Modules/circle_utils.py

```diff
@@ -8,6 +8,10 @@ def get_all_items(relative_url, headers, pagination_limit=5):
     pages_iterated = 0
     while True:
         res = requests.get(temp_url, headers=headers).json()
+        if (res.get("message") == 'An internal server error occurred.'):
```

Comment:
Does this not get caught below in the try catch? Maybe extra context there would suffice?

Reply:
so yes... and it gets caught and rethrown, and the exception thrown exits the program.

I don't know if it's a good idea to squash the error - forcing the pagination iteration to be over - or figure out why exactly in might be sad (or maybe it was just cranky yesterday afternoon) or write things in a more streaming fashion so an error won't throw away progress